magicctapipe/scripts/lst1_magic/config.yaml

```diff
@@ -175,65 +175,60 @@ event_classifier:
         max_samples: null
 
     features: [
-        'intensity',
-        'length',
-        'width',
-        'skewness',
-        'kurtosis',
-        'slope',
-        'intensity_width_2',
-        'h_max',
-        'impact',
+        "intensity",
+        "length",
+        "width",
+        "skewness",
+        "kurtosis",
+        "slope",
+        "intensity_width_2",
+        "h_max",
+        "impact",
     ]
 
     gamma_offaxis:
-        min: 0.2   # unit: [deg]
-        max: 0.6   # unit: [deg]
+        min: null
+        max: null
 
 
 create_irf:
     quality_cuts: null
-    irf_type: 'software'   # select 'hardware', 'software', 'software_with_any2' or 'magic_stereo'
-    dl2_weight: null   # type of the weight for averaging tel-wise DL2 parameters, null, 'var' or 'intensity'
-    obs_time: 50   # used for weighting background events, unit: [hour]
+    irf_type: "software"  # select "software(_only_3tel)", "magic_only" or "hardware"
```

Comment:
this (_only_3tel) in the comment, is a bit confusing, best to list each separately, software, software_only_3tel, ...

Reply:
Thanks, I changed it.